x-pack/plugins/apm/public/components/app/transaction_details/trace_samples_tab.tsx

```diff
@@ -63,7 +59,6 @@ function TraceSamplesTab({
             urlParams={urlParams}
             waterfall={waterfall}
             isLoading={waterfallStatus === FETCH_STATUS.LOADING}
-            exceedsMax={exceedsMax}
```

Comment:
This was moved to `waterfall` object to avoid passing it separately through many intermediate components.